akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -733,12 +732,20 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
    * This operation demultiplexes the incoming stream into separate output
    * streams, one for each element key. The key is computed for each element
    * using the given function. When a new key is encountered for the first time
-   * it is emitted to the downstream subscriber together with a fresh
-   * flow that will eventually produce all the elements of the substream
-   * for that key. Not consuming the elements from the created streams will
-   * stop this processor from processing more elements, therefore you must take
-   * care to unblock (or cancel) all of the produced streams even if you want
-   * to consume only one of them.
+   * a new substream is opened and subsequently fed with all elements belonging to
+   * that key.
+   *
+   * The object returned from this method is not a normal [[Flow]],
+   * it is a [[SubFlow]]. This means that after this combinator all transformations
+   * are applied to all encountered substreams in the same fashion. Substream mode
+   * is exited either by closing the substream (i.e. connecting it to a [[Sink]])
+   * or by merging the substreams back together; see the `to` and `mergeBack` methods
+   * on [[SubFlow]] for more information.
+   *
+   * It is important to note that the substreams also propagate back-pressure as
+   * any other stream, which means that blocking one substream will block the `groupBy`
+   * operator itself&mdash;and thereby all substreams&mdash; once all internal or
```

Comment:
let's just say "ancient, no longer supported browsers" instead of "old browsers" ;) I don't think we need to worry about that. If someone reports this problem we will fix it, but I highly doubt that this will be a problem nowadays.